.github/workflows/user_yaml_ci.yaml

```diff
@@ -41,8 +41,8 @@ jobs:
         python-version: 3.9
     - name: Install dependencies
       run: |
+        pip install --upgrade pip
         pip install gen3users
-        pip install pyyaml==5.4
```

Comment:
since we use it explicitly in the linting step, i would rather we keep installing it explicitly even if at the moment gen3users already includes it. We can remove the pin though